scripts/dev_tools/extract_pr_comments.py

```
@@ -0,0 +1,326 @@
+import sys
+from datetime import datetime, timezone
+from typing import Any
+
+from comment_formats import to_fixer, to_jsonl
+
+
+def run_gh_command(args: list[str], json_output: bool = False) -> tuple[str, int]:
+    """Run a gh command and return output, exit code."""
+    try:
+        result = subprocess.run(
+            ["gh"] + args,
```

Ruff (0.15.17)

[warning] Consider `["gh", *args]` instead of concatenation

Replace with `["gh", *args]`

(RUF005)